pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -42,17 +44,17 @@ CREATE TABLE SCHEMA_CATALOG.label_key(
 );
 
 CREATE TABLE SCHEMA_CATALOG.label_key_position (
-    metric text,
+    metric_name text, --references metric.metric_name NOT metric.id for performance reasons
     key TEXT, --NOT label_key.id for performance reasons.
     pos int,
```

Comment:
I'm a bit concerned with running out of label positions, especially since I don't know a good way to GC them. Should we switch these to a `BIGINT`? It's won't really help now, but we can switch to a custom array-type than can handle them...

Reply:
I think if we have an array that big we are in real trouble. fwiw we see sizes of mostly < 20 now. I'd rather push this off

Reply:
Agreed, if we have arrays the size of int we're screwed.